SCYLLA-VERSION-GEN

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRODUCT=scylla
-VERSION=666.development
+VERSION=x.y.dev
```

Comment:
this would break the install of the RPM, this one need to match what's on master exactly.

Reply:
Why are we using x.y.dev instead of 4.4.dev?  the change first need to be merged in Scylla.

Reply:
updated to 4.4.dev